backend/controller/ingress/handler.go

```diff
@@ -35,13 +37,15 @@ func Handle(
 			http.NotFound(w, r)
 			return
 		}
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+		logger.Errorf(err, "failed to resolve route for %s %s", r.Method, r.URL.Path)
+		http.Error(w, internalServerError, http.StatusInternalServerError)
 		return
 	}
 
 	body, err := BuildRequestBody(route, r, sch)
 	if err != nil {
-		http.Error(w, err.Error(), http.StatusBadRequest)
+		// Don't log on bad request, this can be trivially triggered by malicious clients
```

Comment:
I think it would be worth logging, but at Debug/Trace, so that it's useful during local development.

Reply:
I have added it at debug